tests/5.1/flush/test_flush_seq_cst.c

```diff
@@ -0,0 +1,45 @@
+//===---- test_flush_seq_cst.c ----------------------------------------------===//
+// 
+// OpenMP API Version 5.1 Nov 2020
+// 
+// This is a test of the flush directive with no memory-order-clause specified.
+// Additionally, atomic constructs are used alongside the explicit flush directives
+// to specify memory ordering amongst the two threads.
+//
+// Based on OpenMP 5.0 Example aquire_release.3.c
+// 
+// OpenMP 5.1 states that "If neither memory-order-clause nor a list appears on 
+// the flush construct then the behavior is as if memory-order-clause is seq_cst,"
+// so tests is based on test_flush_no_memory_order_clause.c
+//===------------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+int errors = 0;
+
+int main() {
+
+   int x = 0, y = 0;
+   #pragma omp parallel num_threads(2)
+   {
+      int thrd = omp_get_thread_num();
+      if (thrd == 0) {
+         x = 10;
+         #pragma omp flush seq_cst
```

Comment:
remove seq_cst

Reply:
I'm a bit confused.. the test is for flush seq_cst

Reply:
seq_cst  is the memory order clause. I thought you were checking if the default behavior is the same as if seq_cst was specified.